README.md

```diff
@@ -393,6 +393,8 @@ export const Jumbotron = ({ text }) => (
 
 You can be notified when elements managed by the virtual DOM are created, updated or removed via lifecycle events. Use them for animation, data fetching, wrapping third party libraries, cleaning up resources, etc.
 
+Note that lifecycle events are attached to DOM nodes not to components.  Consider adding a key to ensure that the event is attached to a specific DOM item, rather than a recycled one.
```

Comment:
“DOM nodes” should be virtual-DOM nodes, and “DOM items” should be “DOM elements”

Reply:
Other than that, I like it!